src/index.html

```diff
@@ -12,36 +12,46 @@
     <script src='https://kit.fontawesome.com/a076d05399.js' crossorigin='anonymous'></script>
 </head>
 <body>
-    <h1 class="text-dark mx-5 my-4">Leaderboard</h1>
-    <main class="container gap-5 main d-md-flex flex-row">
-        <!-- score -->
-        <div class="score gap-3 d-flex flex-column">
-            <div class="head gap-3 d-flex flex-row">
-                <h3>Recent scores</h3>
-                <button type="button" class="btn btn-light shadow refresh">Refresh</button>
-            </div>
-            
-            <table class="table table-striped">
-                <tbody class="table-body">
-                </tbody>
-            </table>
+    <main class="main d-flex flex-column">
+        <div class="seven d-flex justify-content-center">
+            <h1>Leaderboard</h1>
         </div>
-        <!-- form for inputting score -->
-        <div class="add-score">
-            <h3>Add your score</h2>
-            <form class="d-flex flex-column form1">
-                <div class="col-md-12">
-                    <label for="name" class="form-label">Name</label>
-                    <input type="text" class="form-control name" id="name">
-                </div>
-                <div class="col-md-12">
-                    <label for="number" class="form-label">Score</label>
-                    <input type="number" class="form-control score" id="number">
-                </div>
-                <div class="col-12 mt-3">
-                    <button type="submit" class="btn btn-primary bg-dark shadow submit">Submit</button>
+        <div class="container mt-4 p-4 gap-5 content d-md-flex flex-row bg-dark shadow-lg">
+            <!-- score -->
+            <div class="score gap-3 d-flex flex-column">
+                <div class="head gap-5 d-flex flex-row justify-content-between">
+                    <h3 class="text-white">Recent scores</h3>
+                    <button type="button" class="btn btn-primary bg-dark shadow refresh">Refresh</button>
                 </div>
-            </form>
+                
+                <table class="table table table-striped table-dark">
+                    <thead>
+                        <tr>
+                          <th scope="col">Player</th>
+                          <th scope="col">Score</th>
+                        </tr>
+                     </thead>
+                    <tbody class="table-body">
+                    </tbody>
+                </table>
+            </div>
+            <!-- form for inputting score -->
```

Comment:
- [ ] [OPTIONAL] You can consider adding some kind of form validation to prevent long scores from sending to the API, the long inputs can break your app UI

![Screenshot_20220616_230746](https://user-images.githubusercontent.com/25227022/174184918-ad3f284b-47dc-4722-9984-0e6b9893e117.png)

Reply:
Thanks for your CR. I am going to work right now on these optional required changes.